src/inbox/mod_inbox_rdbms_pgsql.erl

```diff
@@ -49,13 +50,12 @@ set_inbox(Username, Server, ToBareJid, Content, Count, MsgId, Timestamp) ->
 set_inbox_incr_unread(Username, Server, ToBareJid, Content, MsgId, Timestamp) ->
     mongoose_rdbms:sql_query(Server,
         ["insert into inbox(luser, lserver, remote_bare_jid, "
-                           "content, unread_count, msg_id, timestamp) "
+                            "content, unread_count, msg_id, timestamp) "
                 "values (", esc_string(Username), ", ", esc_string(Server), ",",
                             esc_string(ToBareJid), ", ", esc_string(Content), ", ", "1", ", ",
                             esc_string(MsgId), ", ", esc_int(Timestamp), ") "
             "on conflict (luser, lserver, remote_bare_jid) do "
             "update set content=", esc_string(Content), ", "
-                       "unread_count=inbox.unread_count + 1, "
-                       "msg_id=", esc_string(MsgId), ", ",
-                       "timestamp=", esc_int(Timestamp), ";"]).
-
+                     "unread_count=inbox.unread_count + 1, "
+                     "msg_id=", esc_string(MsgId), ", ",
+                     "timestamp=", esc_int(Timestamp), "returning unread_count;"]).
```

Comment:
I like how elegant it is with PgSQL to read the updated value in one query! It's a pity we don't have the same for MySQL and MSSQL.